frontend/src/hooks/useSessionValid.js

```diff
@@ -50,6 +50,7 @@ function useSessionValid() {
       const setOrgRes = await axios(requestOptions).catch((error) => {
         if (error?.response && error?.response?.status === 403) {
           navigate("/", { state: null });
+          window.location.reload();
```

Comment:
@jagadeeswaran-zipstack Since we're performing a reload, I think it would be safe to remove the navigation part. Please test this change and push the necessary updates.